Ushanka Trinket Addition - Branch

PathOwOgen opened the pull request:
## About the PR
This is a PR in order to add the Ushanka hat to the Trinkets list for Characters.

## Why / Balance
I believe specialty hats is a lovely thing for Trinkets, and currently AFAIK, Ushankas are unable to be ordered or made. but rather simple found around places like Maints. With Ushankas (and potentially more hats later) added to the Trinkets list, this item will be much more likely to be used, and adds more variety to the choices in Trinkets. As for Balance, Ushankas are purely a cosmetic for characters so I do not think there is any concerns for balance, other than the rarity of Ushankas being found in vents being suppressed.

## Technical details

Changes have been done to the Loadout Groups yml, and the Trinkets yml in order to facilitate this change. in total 7 lines of code has been added. 

## Media
![UshankaInMenu](https://github.com/user-attachments/assets/7415486c-ba83-4b9d-9f34-56683f9b5278)
Screenshot of the Ushanka in the Loadouts menu.
## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: 
- add: Added Ushankas to the Trinkets list, allowing characters to take a Ushanka as a Round Start item as a Trinket.

PathOwOgen committed:
Update to trinkets.yml Ushanka

An update the trinkets.yml list for character's loadouts.

Adds the Ushanka to the possible trinkets list, for more variation on roundstart, While also setting a precedent that hats can be a potential trinket, which will hopefully lead to yet more variety, and hopefully an incentive for more people to add hats.

Signed-off-by: PathOwOgen <95729120+PathOwOgen@users.noreply.github.com>

PathOwOgen committed:
Update trinkets.yml typo

Capitalization

Signed-off-by: PathOwOgen <95729120+PathOwOgen@users.noreply.github.com>

PathOwOgen committed:
Update loadout_groups.yml Added to Loadout Groups under Delta-V Additions

Signed-off-by: PathOwOgen <95729120+PathOwOgen@users.noreply.github.com>

deltanedas commented on Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:
```diff
@@ -367,3 +367,11 @@
+    id: Ushanka
+    storage:
+    back:
+    - Ushanka
```
wrong indentation + move this into the file under _DV

PathOwOgen replied:
Done! Newest commit is to remove it from the normal folder and into the loadouts file under _DV, as well the indentation should be fixed.

I think the ID is wrong so I will need to doublecheck the ID of the Ushanka, but this fixed almost all of the issues that was present before. Thank you for taking the time to find this and taking the time to share with me the solution to this error!

PathOwOgen committed:
Merge branch 'master' into Ushanka-Trinket

PathOwOgen committed:
Requested Change 1

Fixed Indentation, and Removed from loadouts/miscellaneous/trinkets.yml, and added to _DV/loadouts/miscellaneous/trinkets.yml

PathOwOgen committed:
Edit from ClothingHeadHatUshanka to Ushanka

This changes ClothingHeadHatUshanka to Ushanka, this is to hopefully resolve the issue of the Ushanka not appearing in the Loadouts menu, citing as a mishmash in the ID of the loadout in the Trinkets file and the naming of the addition in the loadout_groups file would likely cause an error.

PathOwOgen committed:
Revert "Edit from ClothingHeadHatUshanka to Ushanka"

This reverts commit a83ff77428e1dad06516bad94ff982e425778ac6.

PathOwOgen committed:
Ushanka to ClothingHeadHatUshanka in Trinkets.yml

Changing loadout_groups from ClothingHeadHatUshanka to Ushanka in the Loadout_Groups file caused an error which made the game crash upon opening the loadouts menu. This commit Inverts the change, as an Attempt to fix this, and hopefully, resolve a mishmash that could be causing the Ushanka to not appear in the Loadouts menu.

PathOwOgen committed:
Revert "Ushanka to ClothingHeadHatUshanka in Trinkets.yml"

This reverts commit 8ff8b875cdf816d447fe7a266c92643653ea2dc4.

PathOwOgen commented:
Since the last commit, I have attempted to change line 58 in loadout groups from ClothingHeadHatUshanka to just Ushanka to match lines 36-41 in the trinkets file. This resulted in a game crash during testing upon pressing the loadout button in the customization window.

After this, I tried the inverse, and changed the references to Ushanka to ClothingHeadHatUshanka through lines 36-41 in the trinkets file. This resulted in the same game crash, and was Reverted.

Very strange. I am currently still finding out how to resolve this issue.

Checking a previous pull request, Pull request #2095 , I do not see any differences from that Pull Request, and This one. Which leads me to believe that the Ushanka ID is something i'm missing, I'm looking into the issue.

Also, The previous commit suggested by deltanedas did in fact, Fix the main issue I have encountered before, the Issue of every single other loadout breaking. However the Ushanka is still missing from the Trinkets list. If someone has any suggestions feel free to send them! Thank you for taking the time to read this comment.
![image](https://github.com/user-attachments/assets/f3228b43-c0bf-42ec-805e-7adbc1ed5f97)

PathOwOgen committed:
Ushanka in Trinkets.yml ClothingHeadHatUshanka

An attempt to fix the current issue of the Ushanka not showing up ingame in the loadouts menu by changing the mention of Ushanka in the Trinkets.yml to ClothingHeadHatUshanka, which is mentioned in other files as ClothingHeadHatUshanka, not Ushanka

PathOwOgen committed:
Merge branch 'master' into Ushanka-Trinket

PathOwOgen reviewed (commented):
Request Approved, Changes Made! See attached
![seeattachedchangedfiles](https://github.com/user-attachments/assets/e0d24877-dda3-4e3b-bcb9-3da4cc3c7bb6)

PathOwOgen committed:
Merge branch 'master' into Ushanka-Trinket

deltanedas commented on Resources/Prototypes/_DV/Loadouts/Miscellaneous/trinkets.yml:
```diff
@@ -33,3 +33,9 @@
+  id: ClothingHeadHatUshanka
+  storage:
+    back:
+    - ClothingHeadHatUshanka
```
fix not having a trailing newline

PathOwOgen replied:
Doing right now, thank you!

PathOwOgen replied:
Done, Fixed with commit bd8aa43!

PathOwOgen committed:
Adds a Trailing line

Adds an empty line to the bottom of the trinkets.yml at deltanedas's request

PathOwOgen committed:
Merge branch 'Ushanka-Trinket' of https://github.com/PathOwOgen/Delta-v-PathOwOgen-LocalAvali-Enviroment into Ushanka-Trinket

PathOwOgen committed:
Revert "Adds a Trailing line"

This reverts commit c8a5a38c62e9c61737e5aaa8bdbb70a4a287f2a0.

PathOwOgen committed:
Trailing Line at end of Trinkets.yml

Fixed the previous commit, I added a new line but it wasn't empty, this fixes that. Should now be in line with the requested change.

PathOwOgen committed:
Hopefully added the new line.

I'll admit im new to this whole github thing, but it really shouldn't have taken me three commits to add a trailing line, Holy shoot. Here goes, please cooperate Mr/Mrs Github, may thy git be on my side

PathOwOgen committed:
Revert "Hopefully added the new line."

This reverts commit 1bcc448ec9879774f81ae905b3787fe110fe3867.
